packages/connections/src/components/connections.tsx

```diff
@@ -84,54 +85,56 @@ function Connections({
   } = state;
 
   return (
-    <div
-      data-testid={
-        isConnected ? 'connections-connected' : 'connections-disconnected'
-      }
-      className={connectStyles}
-    >
-      <ResizableSidebar
-        activeConnectionId={activeConnectionId}
-        connections={connections}
-        createNewConnection={createNewConnection}
-        setActiveConnectionId={setActiveConnectionById}
-        onConnectionDoubleClicked={connect}
-        removeAllRecentsConnections={removeAllRecentsConnections}
-        removeConnection={removeConnection}
-        duplicateConnection={duplicateConnection}
-      />
-      <div className={connectItemContainerStyles}>
-        {storeConnectionError && (
-          <Banner
-            variant={BannerVariant.Danger}
-            dismissible
-            onClose={hideStoreConnectionError}
-          >
-            {storeConnectionError}
-          </Banner>
-        )}
-        <div className={formContainerStyles}>
-          <ConnectForm
-            onConnectClicked={(connectionInfo) =>
-              connect({
-                ...connectionInfo,
-              })
-            }
-            onSaveConnectionClicked={saveConnection}
-            initialConnectionInfo={activeConnectionInfo}
-            connectionErrorMessage={connectionErrorMessage}
-          />
-          <FormHelp />
+    <LeafyGreenProvider>
```

Comment:
I think we should add it at the root of the application as the documentation suggests:

> LeafyGreen Provider is a package made to provide, and make it easy to consume **page-wide**, interaction-related state

We also have a [JIRA ticket for that](https://jira.mongodb.org/browse/COMPASS-4986) if you want to attach it to this work